workflows/MODEL_TO_DATA.nf

```diff
@@ -17,10 +19,12 @@ assert params.email_with_score in ["yes", "no"], "Invalid value for ``email_with
 params.cpus = "4"
 // Default Memory to dedicate to RUN_DOCKER
 params.memory = "16.GB"
-// Scoring Script
-params.scoring_script = "model_to_data_score.py"
-// Validation Script
-params.validation_script = "validate.py"
+// The container that houses the scoring and validation scripts
+params.challenge_container = "sagebionetworks/synapsepythonclient:v4.0.0"
+// The command used to execute the Challenge scoring script in the base directory of the challenge_container: e.g. `python3 path/to/score.py`
+params.execute_scoring = "python3 /usr/local/bin/score.py"
```

Comment:
The `pegs-evaluation` Dockerfile [copies the val/score scripts into `/usr/local/bin`](https://github.com/Sage-Bionetworks-Challenges/pegs-evaluation/blob/f3923663cf8aed0e7099913ffaec35859d6a98a5/Dockerfile#L3) so I modified the path to the scripts accordingly, so we don't have to deal with relative paths.